src/java.base/share/classes/sun/util/calendar/ImmutableGregorianDate.java

```diff
@@ -146,18 +145,10 @@ public boolean isNormalized() {
         return date.isNormalized();
     }
 
-    public boolean isStandardTime() {
-        return date.isStandardTime();
-    }
-
     public boolean isDaylightTime() {
         return date.isDaylightTime();
     }
 
-    protected void setLocale(Locale loc) {
-        unsupported();
-    }
-
```

Comment:
This removal does not look right. The class claims `immutable`, and yet it is now allowing setting the locale.

Reply:
There is no `setLocale` method in the base class anymore. So, it's not possible to set locale in any classes which extend `BaseCalendar.Date`, including `ImmutableGregorianDate`

Reply:
It is ok then. (missed the removal of the method in the abstract class). Now come to think of it, I would like them to be `sealed` to make sure they are not randomly subclassed, but that clean-up is for another day.